web/src/composebox_typeahead.ts

```diff
@@ -1363,6 +1325,49 @@ export function initialize({
         },
         stopAdvance: true, // Do not advance to the next field on a Tab or Enter
     });
+}
+
+export function initialize({
```

Comment:
Try keeping `initialize` above `initialize_pm_typeahead`, I think it'll make this diff a lot smaller and easier to read.